accessibility.html

```diff
@@ -13,39 +13,34 @@ <h1>Accessibility</h1>
 
 <h2>Covid</h2>
 
-As of 2023-05-24:
+As of 2023-06-07:
 
 <ul>
 
-<li><p>Everyone at the dance must be boosted, if eligible.
-<ul>
-<li>If you're not yet eligible, due to age or time
-    since your previous shot, that's fine.
-<li>Boosters beyond the first are not required.
-<li>We check cards at the door (photos ok).
-<li>If your doctor has advised you not to get some or all of your
-    shots, or not get them yet, that's ok. Please either email your
-    doctor's note to bida@bidadance.org or bring it with you to the
-    dance.
-</ul>
+<li><p>We host both <strong>mask-required</strong> and <strong>mask-optional</strong> dances.
+The policy for each dance will be announced in advance on the website, email list, and social media.
 
-<li><p>Please notify us after if you learn you may have been
-infectious at the dance.  Without giving your name we'll let the
-community know by posting on Facebook, updating our website, and
-emailing everyone who asked to be notified.
-
-<li><p>Everyone age two and up must wear a <a
+<li><p>At mask-required dances, everyone age two and up must wear a <a
 href="https://blog.bidadance.org/2022/05/requiring-high-filtration-masks.html">high-filtration</a>
 (N95, KN95, KF94, etc) mask, except when (a) actively drinking or
-(b) performing on stage.  If you need to eat, please do so outside.
+(b) performing on stage.
 
-</ul>
+<li><p>Family dances and other afternoon special events will follow the
+same masking policy as the evening dance they precede. Other special
+events will have their own policies which will be announced in advance.
 
-<p>
+<li><p>We’ve resumed potlucks between afternoon and evening dances.
+People are welcome to eat outside or inside. At mask-required events
+please wear your mask when you’re not eating.
+
+<li><p>We will provide high-filtration masks at the door for all dances.
 
-As of 2023-05-21 we've resumed our potlucks.  People are welcome to
-eat outside or inside.  If you're eating inside, please wear your mask
-when you're not eating.
+<li><p>Please notify us after if you learn you may have been
+infectious at the dance.  Without giving your name we’ll let the
+community know by posting on the website and emailing our
+<a href="https://www.bidadance.org/join">covid notification list</a>.
+
+</ul>
```

Comment:
I think we should probably say we no longer require vaccination, so people looking for a vaccination requirement know they can stop searching?

Reply:
Oh right, we also did say we were going to recommend people stay up to date

Reply:
Done!